IonDotnet/Internals/Text/RawTextReader.cs

```diff
@@ -704,8 +704,8 @@ public IEnumerable<SymbolToken> GetTypeAnnotations()
                         throw new UnknownSymbolException(a.Sid);
                     }
 
-                    var text = symtab.FindKnownSymbol(a.Sid);
-                    yield return new SymbolToken(text, a.Sid);
+                    var text = symtab.FindKnownSymbol(a.ImportLocation.Sid);
```

Comment:
One last thing, and then this is good to go:
Line 699 should be `if (a.Text is null && a.ImportLocation != null)` since we're relying on ImportLocation now and not local SID.

Reply:
Did you mean `if (a.Text is null && a.ImportLocation != default)`? Or is `null` what we are checking for in this instance?

Reply:
We want to know if `a.ImportLocation` is specified. If `a.ImportLocation != default` is the correct way to do that, then yes that's what I mean.

Reply:
Done.